lua/kubectl/views/fallback/init.lua

```diff
@@ -77,7 +84,10 @@ end
 --- Get current seletion for view
 ---@return string|nil
 function M.getCurrentSelection()
-  return tables.getCurrentSelection(2, 1)
+  if definition.namespaced then
```

Comment:
This would only work for the CRDS that have that variable right?

Reply:
no, I added it to every resource here:
https://github.com/Ramilito/kubectl.nvim/pull/235/files#diff-4f77b793ea3926dba9cc1fa057d7e215c1ba56312288dca56a4519c1e903066bR116